### PR TITLE
Version update

### PR DESCRIPTION
**Pull request by xoxys**

### CODE REVIEW

**Comment by tboerger** [comment]
Please do not remove the old version directories.

**Comment by xoxys** [comment]
ARM build failure seems to be related to https://github.com/minio/minio/issues/9003

File: README.md [hunk 1]
```diff
@@ -9,10 +9,13 @@ These are docker images for [Minio](https://github.com/minio/minio) running on a
 ## Versions
 
 * [latest](./latest) available as `webhippie/minio:latest`
-* [2020-02-27](./2020-02-27) available as `webhippie/minio:2020-02-27`
-* [2020-01-16](./2020-01-16) available as `webhippie/minio:2020-01-16`
-* [2019-12-19](./2019-12-19) available as `webhippie/minio:2019-12-19`
-* [2019-10-12](./2019-10-12) available as `webhippie/minio:2019-10-12`
+* [latest](./2020-02-27) available as `webhippie/minio:2020-02-27`
```
Comment (tboerger) [inline README.md]: The link names are all latest

Comment (xoxys) [reply README.md]: fixed

**Comment by tboerger** [comment]
Until upstream fixes arm32 support RELEASE.2020-02-07T23-28-16Z seems to be the latest tag we are able to build for all architectures.

**Pull request merged**